scripts/opentitan/otptool.py

```diff
@@ -50,10 +53,35 @@ def parse_lc_token(tkdesc: str) -> tuple[str, 'LifeCycleTokenPair']:
     return token_name, tkeng.build_from_text(tktext)
 
 
+def get_top_name(*filepaths: list[Union[str, IOBase]]) -> Optional[str]:
+    """Try to retrieve the top name from a list of configuration files.
+
+       :param filepaths: list of file path or file objects
+       :return: the name of the identified top, if found
+    """
+    for filepath in filepaths:
+        if not isinstance(filepath, str):
+            filepath = getattr(filepath, 'name', None)
+            if not filepath:
+                continue
+        if not isfile(filepath):
+            continue
+        fdir = dirname(filepath)
+        top_names = set(OpenTitanTop.names)
+        while fdir:
+            fdir, tail = splitpath(fdir)
+            if tail.startswith('top_'):
+                candidate = tail[4:]
```

Comment:
Small nit:

I prefer this because it is more explicit and avoids potential bugs being introduced when updating the string but not the length.
```suggestion
            top_prefix = 'top_'
            if tail.startswith(top_prefix):
                candidate = tail.removeprefix(top_prefix)
```

Reply:
You are right. I though removeprefix had been added later in Python releases, but it is part of 3.9 so perfectly fine to use it.

Reply:
I've propagatd this change to other files as well, but this will be part of the next PR :-)